types/react-native/test/init-example.tsx

```diff
@@ -19,17 +19,21 @@ import {
 
 const App = () => {
     const hasHermes = typeof HermesInternal === 'object' && HermesInternal !== null;
+
+    const isFabricEnabled = Boolean(typeof nativeFabricUIManager);
```

Comment:
This seems like a pretty bad example.  This doesn't tell you if the UI here is using fabric.  The fabric UI manager is a global thing, whereas each rootview can be fabric individually.

Reply:
Okay, I found it as a way to determine whether the fabric is enabled or not. 
You can find it in [example of the library with backward compatibility](https://github.com/react-native-community/RNNewArchitectureLibraries/blob/feat/back-fabric-component-070/colored-view/src/index.js).
Can you suggest any other way to determine whether the fabric is enabled?

Reply:
In that example doesn't the fabric JS work in paper too?

Reply:
No, when the fabric is disabled in the target project this fabric UI manager is undefined so I assume the fabric doesn't work at all

Reply:
I worked with Meta to get the sample changed.  The fabric check isn't needed.

Reply:
Okay, what about commands? 
How we should know which method to use?

Reply:
The JS code for fabric vs non-fabric should work no matter if you are running paper of fabric.  You should use the new methods for commands in all cases.

Reply:
okay, gotcha lets close this one then